modules/understanding-upgrade-channels.adoc

```diff
@@ -7,17 +7,129 @@
-Channels are tied to a minor version of {product-title}. For instance,
-{product-title} {product-version} channels will never include an upgrade to a
-4.4 release. This ensures administrators make an explicit decision to upgrade to
-the next minor version of {product-title}. Channels only control updates and
+= {product-title} Upgrade Channels and Update Types
+
+In {product-title} 4.1, Red Hat introduced the concept of Upgrade Channels for
+recommending the appropriate upgrade versions to your cluster. Upgrade Channels
+separate upgrade strategies and are also used to control the cadence of updates.
+Upgrade Channels are tied to a minor version of {product-title}. For instance,
+{product-title} {product-version} Upgrade Channels will never include an upgrade to a
+4.5 release. This ensures administrators make an explicit decision to upgrade to
+the next minor version of {product-title}. Upgrade Channels only control updates and
 have no impact on the version of the cluster you install; the
 `openshift-install` binary for a given patch level of {product-title} always
 installs that patch level.
 
-See link:https://access.redhat.com/articles/4495171[OpenShift 4.2 Upgrades phased roll out] for more information on the types of updates and upgrade channels.
+The updates within an Upgrade Channel are designated as one of two types. These
+Update Types are defined as:
+
+.  General Availability (GA) Software - {product-title} GA releases are fully
+supported and production quality.
+
+.  Release Candidate (RC) Software - {product-title} RC releases are not supported
+and should only be used to test feature acceptance and assist in qualifying the next
+version of {product-title} in your infrastructure. RC releases contain all the
+features of the product and are representative of an eventual GA release.
+
++                                                                                    
+[NOTE]                                                                               
+==== 
+RC releases differ from the nightly builds found on try.openshift.com. You
+cannot upgrade nightly builds to nightly builds. Nightly builds are available for
+early access to features but are not upgradable or supported.
+==== 
+
+{product-title} {product-version} has three Upgrade Channels to choose from:
+
+* candidate-{product-version}
+* fast-{product-version}
+* stable-{product-version}
+
+RC release versions are only available in candidate-{product-version}.
+
+GA release versions are only availble in fast-{product-version} or stable-{product-version}.
+
+More details about each channel and their usage are in the following sections.
+
+= candidate-{product-version}
+
+The candidate channel allows upgrading from an RC version to a later RC version and
+upgrading from a previous minor version of {product-title} to the current RC version.
+You cannot upgrade from an RC release to a GA release of {product-title}.
```

Comment:
This is misleading, because you may also be able to update to GA from an RC release in the candidate channel.  See https://github.com/openshift/openshift-docs/pull/18944#discussion_r365289359

Reply:
Suggested rewording:

> The candidate-{product-version} channel is the first {product-version} channel to which releases are promoted.  If a release performs well as a candidates [would be nice to link to Telemetry-reporting section here], it is later promoted to fast-{product-version} [would be nice to link to the fast section here], but never removed from candidate-{product-version}.  candidate-{product-version} may also contain {product-version-minus-one} releases which are being considered as jumping-off points for {product-version-minus-one} to {product-version} updates.  candidate-{product-version} will never contain nightly releases (which will never belong to any channel) or releases from {product-version-plus-one} or later.

Reply:
Couple of comments
1. We should not use `release candidates` instead of `RC`. Reading `RC` might not be user friendly to may folks. 
2. We just merge 
```
{product-title} {product-version} has three Upgrade Channels to choose from:

* candidate-{product-version}
* fast-{product-version}
* stable-{product-version}
```
with this section. I do not think having information in two places going to improve the reading experience but on the contrary it forces the reader to scroll up again and remember what he/she read before.

Reply:
Actually I find @wking's suggested comment has a lot of information which would confuse folks. The initial sections should be quick read for people then we want to extra information we should add them in the later part of the document somehow.

Reply:
> The initial sections should be quick read for people then we want to extra information we should add them in the later part of the document somehow.

Maybe "this is what life in `stable-{product-version}` will be like" (90+% of admins) with later sections for fast and candidate?